web/beacon-app/src/utils/strings.ts

```diff
@@ -5,6 +5,11 @@ export const capitalize = (str: string) => {
 
 export const getInitials = (name: string) => {
   const nameArray = name.split(' ');
-  const initials = nameArray[0].charAt(0) + nameArray[1].charAt(0);
+  const initials =
+    nameArray.length >= 2
+      ? nameArray[0].charAt(0) + nameArray[1].charAt(0)
+      : nameArray[0].charAt(0) + nameArray[0].charAt(1);
+
```

Comment:
This helps prevent potential errors that might occur when the user's full name contains only one word.